keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone_presets.py

```diff
@@ -21,6 +21,9 @@
                 "normalization and hard-swish activation are applied after the "
                 "convolution layers."
             ),
+            "params": 933502,
```

Comment:
lets add underscores:

933_502

Reply:
I don't think this will make any difference to the python interpreter run by keras.io